### PR TITLE
Update README.md _.times implement 

### PR DESCRIPTION
**Pull request by odykyi**
_.times add:

` var result = [...Array(10).keys()]`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1975,10 +1975,15 @@ Invokes the iteratee n times, returning an array of the results of each invocati
   console.log(result)
   // output: '[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]'
 
-  // Native
+  // Native(first variation)
```
Comment (stevemao) [inline README.md]: Looking at other examples, you don't need to mention `(first variation)`

**Comment by stevemao** [comment]
Thanks for the PR!

**Comment by odykyi** [comment]
@stevemao updated!

**Comment by odykyi** [comment]
@stevemao pls merge

**Pull request merged**